includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -82,6 +82,13 @@ class AMP_Validated_URL_Post_Type {
 	 */
 	const VALIDATION_ERRORS_META_BOX = 'amp_validation_errors';
 
+	/**
+	 * The transient key to use for caching the pending URLs count.
```

Comment:
Instead of "pending" it would be more accurate to use "new" for the changes here. The URLs being queried are those with _new_ accepted/rejected validation errors:

```
					AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_NEW_REJECTED_STATUS,
					AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_NEW_ACCEPTED_STATUS,
```

As opposed to _ack_ rejected/accepted errors.

Using `new` instead of `pending` would be more consistent in regards to terminology.

But given the  resulting containing element  is `pending-count`, I don't feel strongly about this.

Reply:
The `pending-count` class name is only appearing here because the Comments admin menu item also uses it, and the markup for it was copied:

![image](https://user-images.githubusercontent.com/134745/64631913-f634c580-d3ac-11e9-95f0-45480683f8ac.png)

I thought that `pending-count` had some styles associated with it, but this is not the case. Feel free to rename the class name as you see fit as well.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3222/commits/b887ee5c4470334d0a03c6fd24b5fc04b8f7569e